### PR TITLE
Ballot app

### PR DESCRIPTION
**Pull request by adborden**

### CODE REVIEW

**Comment by adborden** [comment]
Fixes #4.

File: files/default/deploy-backend [hunk 1]
```diff
@@ -12,9 +12,10 @@ sudo -i -u backend /bin/bash -exc "
   mkdir -p $deploy_dir;
   git archive origin/master | tar xC $deploy_dir;
   source /data/backend/env/bin/activate;
-  pip install -r $deploy_dir/requirements.txt;
+  pip install --exists-action w -r $deploy_dir/requirements.txt;
```
Comment (tdooner) [inline files/default/deploy-backend:15]: Huh, what does `--exists-action` do...?

```
  --exists-action <action>    Default action when a path already exists: (s)witch, (i)gnore, (w)ipe, (b)ackup.
```

Interesting.

File: templates/default/nginx.conf.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@ upstream disclosurebackend {
 
 server {
   listen 80;
-  server_name opencal.opendisclosure.io;
+  server_name opencal.opendisclosure.io disclosure.caciviclab.org;
```
Comment (tdooner) [inline templates/default/nginx.conf.erb:7]: This new hostname will need to be added to settings_local.py's `ALLOWED_HOSTS` as well, but that isn't in version control.

**Comment by tdooner** [comment]
Cool, lgtm.

File: templates/default/nginx.conf.erb (hunk 2 above)
Comment (adborden) [inline templates/default/nginx.conf.erb:7]: Yep, already covered. btw, we should be backing that up I guess. I'll add it to our backup issues.

**Pull request merged**